plugins/auth-backend/src/lib/flow/authFlowHelpers.ts

```diff
@@ -25,16 +25,43 @@ export const postMessageResponse = (
 ) => {
   const jsonData = JSON.stringify(response);
   const base64Data = Buffer.from(jsonData, 'utf8').toString('base64');
+  const base64Origin = Buffer.from(appOrigin, 'utf8').toString('base64');
```

Comment:
SHOULD not be necessary, but you know - it's ultimately user controllable in a way and it doesn't hurt to be thorough

Reply:
Sure xP, It's not user controllable though, only controllable by whoever has access to config, and with that usually the code itself anyway :p

Doesn't hurt though